src/index.js

```diff
@@ -81,17 +129,32 @@
       }
       return el;
     },
+    getElByDataAttribute: (attribute, value, scope = document) => {
+      const el = scope.querySelector(`[data-${attribute}="${value}"]`)
+      return el
+    },
     getAllElByClass: (className) => {
       const elements = document.querySelectorAll(`*[class*="${className}"]`);
       if (!elements) {
         teleport.error(`did not find ${className} class`);
       }
       return elements;
     },
+    getAllElementsByDataAttribute: (attribute, value, scope = document) => {
+      const elements = scope.querySelectorAll(`[data-${attribute}="${value}"]`)
+      if (!elements) {
+        teleport.error(`did not find any elements with data-${attribute}=${value}`);
```

Comment:
I'm not so sure if this must be an error. I think a just a log would be a better option.